woocommerce/woocommerce-gateway-stripe
# chore: add additional payment methods

# Changes proposed in this Pull Request:

Just adding some additional payment method icons from the design library. Nothing much.

# Testing instructions

Nothing much to test, unfortunately - it's just exported from Figma and added to the app.
Nothin is breaking.

If you _really_ want to test it, you could mock these selectors to return the additional ids: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/81efec236386daffb3cce38964cf43a7ab7e5bb3/client/settings/general-settings-section/data-mock.js

Then you'll notice that iDeal might look a little funky (for now - waiting on UX to provide a different asset). We might not launch with iDeal, so it's not a problem if this merges, we can always update it.
![Screen Shot 2021-09-16 at 10 15 15 PM](https://user-images.githubusercontent.com/273592/133723904-9551b0f2-356e-4a55-b34a-f15f15bc0fca.png)
